modules/enable-offline-mode-roxctl.adoc

```diff
@@ -10,21 +10,21 @@ You can enable offline mode when you are installing {product-title} by using the
 
 .Procedure
 
-. If you are using a registry other than the default internet-connected registry (`stackrox.io`), provide the locations where you have pushed the {product-title} images when answering the `image to use` prompts:
+. If you are using a registry other than the default internet-connected registry (`registry.redhat.io`), provide the locations where you have pushed the {product-title} images when answering the `image to use` prompts:
 +
 [source,terminal,subs=attributes+]
 ----
-Enter main image to use (default: "registry.redhat.io/rh-acs/main:{rhacs-version}"): <your_registry>/stackrox/main:{rhacs-version}
+Enter main image to use (if unset, the default will be used): <your_registry>/rhacs-main-rhel8:{rhacs-version}
```

Comment:
If this is still the default image, it's less clear now.

Reply:
Thanks @kalexand-rh These are prompts that the `roxctl CLI` generates. Although I cannot change the prompts, I can confirm with Engineering about the default image.